.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+### Describe the feature request or enhancement you'd like
+A clear and concise description of the enhancements you want, i.e., _"As a skills architect, I want the ability to edit multiple RSDs in bulk so that I don't need to make the same edits to each one individually"_
+
+### Is your feature request related to a problem? Please describe it.
```

Comment:
Given the changes above, this section seems redundant.

Reply:
I see 2 different questions with 2 different answers, one focused on the improvement, and another focused on whatever issue or shortcoming motivated it.